tests/fs_file.rs

```diff
@@ -33,6 +40,17 @@ fn basic_read() {
     });
 }
 
+#[test]
```

Comment:
Might wanna try this with some really big generated array of bytes, possibly with something like quickcheck, just to be sure.

Reply:
@Noah-Kennedy Is this still a concern?